src/main/java/io/codelex/flightplanner/FlightPlannerApplication.java

```diff
@@ -2,8 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
```

Comment:
Kamdēļ šo liki klāt?

Reply:
Neuzmanības kļūda. Mēģināju miljons interneta un ChatGPT ieteiktos variantus, lai panāktu, ka aiziet addFlight testi, bet pēc salabošanas šo neesmu pamanījusi nodzēst.